stacker/providers/aws/interactive.py

```diff
@@ -205,23 +255,27 @@ def __init__(self, region, replacements_only=False, *args, **kwargs):
         self.replacements_only = replacements_only
         super(Provider, self).__init__(region=region, *args, **kwargs)
 
-    def update_stack(self, fqn, template_url, parameters, tags, diff=False,
-                     **kwargs):
+    def update_stack(self, fqn, template_url, old_parameters, parameters,
+                     tags, diff=False, **kwargs):
         changes, change_set_id = create_change_set(self.cloudformation, fqn,
                                                    template_url, parameters,
                                                    tags, **kwargs)
+        params_diff = diff_parameters(
+            AWSProvider.params_as_dict(old_parameters),
+            AWSProvider.params_as_dict(parameters))
 
         action = "replacements" if self.replacements_only else "changes"
         full_changeset = changes
         if self.replacements_only:
             changes = requires_replacement(changes)
 
-        if changes:
-            output_summary(fqn, action, changes,
+        if changes or params_diff:
```

Comment:
Maybe there should be a note if there's a parameter change, but no changeset changes  - just to let people know that it's not an error?  Any ideas on that?